web/src/markdown.ts

```diff
@@ -690,12 +688,13 @@ export function parse({
 
     // No <code> around our code blocks instead a codehilite <div> and disable
     // class-specific highlighting.
-    renderer.code = (code: string): string => fenced_code.wrap_code(code) + "\n\n";
+    renderer.code = (code) => fenced_code.wrap_code(code) + "\n\n";
```

Comment:
Why did you remove these type annotations?

Reply:
Because we don’t need to be explicit here, typescript is able to infer types for us here.
But if you think it’s nice to be explicit then sure.